rust/template/distributed_datalog/src/read.rs

```diff
@@ -0,0 +1,21 @@
+//! A module containing traits that abstract away the retrieval of
```

Comment:
`read_config.rs` might be a better name for this file.

Reply:
Yeah, I was struggling hard to name this file :-0